apps/docs/src/app/core/component-docs/message-toast/message-toast-header/message-toast-header.component.html

```diff
@@ -0,0 +1,25 @@
+<header>Message Toast</header>
+<description
+    >Message Toast is a small, non-disruptive popup for success messages that disappears automatically after a few
```

Comment:
The message toast is a small, non-disruptive popup for success messages that disappears automatically after a few seconds. When message toast is displayed, it is always centered horizontally at the bottom of the screen and fades in and out automatically.

Note: The timing and duration of the message toast is defined by the application; the standard value being 3000 ms. The duration can be above 3000 ms, but it's not recommended.